src/app/components/filebrowseruss/filebrowseruss.component.ts

```diff
@@ -246,18 +246,24 @@ export class FileBrowserUSSComponent implements OnInit, OnDestroy {//IFileBrowse
     let oldPath = file.path;
     file.selectable = false;
     let renameFn = (node: HTMLElement) => {
+      renameField.parentNode.replaceChild(node, renameField);
       file.selectable = true;
       let nameFromNode = renameField.value;
       let pathForRename:any;
       pathForRename = (oldPath as String).split("/");
       pathForRename.pop();
       pathForRename = pathForRename.join('/');
       if(oldName != nameFromNode){
-        this.ussSrv.renameFile(oldPath, `${pathForRename}/${nameFromNode}`).subscribe(
+        let newPath = `${pathForRename}/${nameFromNode}`;
+        this.ussSrv.renameFile(oldPath, newPath).subscribe(
           res => {
             this.snackBar.open(`Renamed: ${oldName} to ${nameFromNode}`,
               'Dismiss', { duration: 5000,   panelClass: 'center' });
```

Comment:
We now have a `zlux-file-explorer\src\app\shared\snackbar-options.ts` btw that has
`defaultSnackbarOptions = { duration: 5000, panelClass: 'center' };`